cmd/keygen/main.go

```diff
@@ -63,27 +76,40 @@ func main() {
 		log.Fatalf("failed to get peer identity from public key: %s", err)
 	}
 
-	// Write keys and identity to files.
-
 	pubKeyFile := filepath.Join(flagOutputDir, pubKeyName)
 	err = os.WriteFile(pubKeyFile, pubPayload, pubKeyPermissions)
 	if err != nil {
-		log.Fatalf("could not write private key to file: %s", err)
+		log.Fatalf("could not write public key to file: %s", err)
+	}
+
+	pubKeyBase64 := base64.StdEncoding.EncodeToString(pubPayload)
+	pubKeyTxtFile := filepath.Join(flagOutputDir, pubKeyTxtName)
+	err = os.WriteFile(pubKeyTxtFile, []byte(pubKeyBase64), pubKeyPermissions)
+	if err != nil {
+		log.Fatalf("could not write public key string to file: %s", err)
 	}
 
 	idFile := filepath.Join(flagOutputDir, identityName)
 	err = os.WriteFile(idFile, []byte(identity), pubKeyPermissions)
 	if err != nil {
-		log.Fatalf("could not write private key to file: %s", err)
+		log.Fatalf("could not write identity to file: %s", err)
+	}
+
+	// Write the Peer ID to a file
+	peerIDFile := filepath.Join(flagOutputDir, peerIDFileName)
+	err = os.WriteFile(peerIDFile, []byte(identity.Pretty()), pubKeyPermissions)
```

Comment:
I'd use `identity.String()` instead of `Pretty()`, since `Pretty()` is deprecated.

Reply:
thanks for the tip!